evaluation.md

```diff
@@ -1,19 +1,30 @@
 # Evaluation: running commentary
 
-## Internal DSL
+## External DSL
 
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
-**On a scale of 1–10 (where 10 is "a lot"), how much did you have to change your syntax?**
+I originally wanted it such that you could have a header which would represent a State, and then the rules after it. See:
 
-**On a scale of 1–10 (where 10 is "very difficult"), how difficult was it to map your syntax to the provided API?**
+State 1:
+if [N: open] -> go N, change to State 1
+if [N: open, S: blocked] -> go S, don't change State 
 
```

Comment:
I like this idea of using english. I also ended up using an english implementation through string conversion but this was much more complicated then it probably needed to be.